docs/source/install.rst

```diff
@@ -65,102 +58,114 @@ According to the `contributors guidelines <contributors_guidelines>`__, you shou
-	
 	.. code-block:: bash
 
-		pip install -r requirements.txt
-	
-	.. warning::
-		Note: In future, when the conda disdrodb feedstock installation will be set up, the following code should be used instead: 
-	
-		.. code-block:: bash
-	
-	     		conda install --only-deps disdrodb
-		
-	* Install disdrodb 
+	   pip install -r requirements.txt
+	   
 	
-	.. code-block:: bash
+	We recomand you to use a virtual environment. This can be done as follow :
 	
-		pip install -e .
 	
-		
-* **Conda (Option 2)**
+	**With venv :**  
 
+		* Windows: Create a virtual environment with venv:
 
-	* Create the `disdrodb-dev` conda environment and install the required dependencies:
+			.. code-block:: bash
 
-	.. code-block:: bash
+			   python -m venv venv
+			   cd venv/Script
+			   activate
+			   pip install -r requirements.txt
+			   
 
-		conda env create -f environment.yml 
+		* Mac/Linux: Create a virtual environment with venv:
+
+			.. code-block:: bash
+
+			   virtualenv -p python3 venv
+			   source venv/bin/activate
+			   pip install -r requirements.txt
+
+	**With Conda env :**
 	
-	.. warning::
-		
-		Note: This command takes quite some time at the moment ... 
+		* Create the `disdrodb-dev` conda environment:
 
-	* Activate the disdrodb conda environment
+			.. code-block:: bash
+			
+				conda create --name disdrodb-dev python=3.9 --no-default-packages
 
-	.. code-block:: bash
+		* Activate the disdrodb conda environment:
+
+			.. code-block:: bash
 
-		conda activate disdrodb-dev
+				conda activate disdrodb-dev
 		
-	* Manually add the path of your local copy of disdrodb to the ``PYTHONPATH`` environment variable. 
-	  In Linux operating systems, you could add the following line to your ``.bashrc`` file located in the ``/home/<your_username>`` directory: 
-	  
-	  .. code-block:: bash
-	  	export PYTHONPATH="${PYTHONPATH}:/path/to/your/local/repo/of/disdrodb/"
+		* Check you are inside the disdrodb local repository, and install the required dependencies with:
+		
+			.. code-block:: bash
 
+				pip install -r requirements.txt
+			
+			.. warning::
+				Note: In future, when the conda disdrodb feedstock installation will be set up, the following code should be used instead: 
+			
+				.. code-block:: bash
+			
+					conda install --only-deps disdrodb
+			
+		* Install disdrodb 
+		
+			To install the project in editable mode : 
+		
+			.. code-block:: bash
+			
+				pip install -e .
+	
+	
 
-* **Pip**
 
-	* (Optional) We recommend you install in a virtual environment, for example with venv:
 
-		* Windows: Create a virtual environment with venv:
+	
+		
+* **Conda**
 
-			.. code-block:: bash
 
-			   python -m venv venv
-			   cd venv/Script
-			   activate
+	* Create the `disdrodb-dev` conda environment and install the required dependencies:
 
-		* Mac/Linux: Create a virtual environment with venv:
+		.. code-block:: bash
 
-			.. code-block:: bash
+			conda env create -f environment.yml 
```

Comment:
This is the alternative to using `pip install requirements.txt` 
I would move it above, just before or above the note on `conda install --only-deps disdrodb`

Reply:
@ghiggi We had a discussion on that with @charlottegiseleweil. If you don't mind, I close thisPR and we discuss that on monday morning together.